contrib/mesos/pkg/scheduler/podtask/pod_task.go

```diff
@@ -33,8 +33,8 @@ import (
 )
 
 const (
-	DefaultContainerCpus = 0.25 // initial CPU allocated for executor
-	DefaultContainerMem  = 64   // initial MB of memory allocated for executor
+	DefaultContainerCpus = 0.25                   // initial CPU allocated for executor
+	DefaultContainerMem  = 64.0 * 1024.0 * 1024.0 // initial MB of memory allocated for executor
```

Comment:
The resource type is only helpful when parsing and printing quantities. It does not support help with conversions.